examples/impls/test/time-sync-avoid-padding.yml

```diff
@@ -0,0 +1,50 @@
+name: nesting-demo
+description: impl with 2 levels of nesting with non-uniform timing of observations, with flag to avoid time-sync by erroring out
+tags:
+initialize:
+  models:
+    - name: time-synchronization
+      model: TimeSyncModel
+      path: builtin
+      config: 
+        start-time: 2023-12-12T00:00:00.000Z # ISO timestamp
+        end-time: 2023-12-12T00:01:00.000Z # ISO timestamp
+        interval: 5 # seconds
+        error-on-padding: true
```

Comment:
@narekhovhannisyan @pazbardanl minor point, but interested to hear your opinion on the field name. I'd suggest the name should be more positive action oriented, such as `allow-padding` and having `true` being the enabler and `false` triggering the exception, rather than `true` being the error case.
(not at all blocking)

Reply:
agree. it's more intuitive and requires less 'double-negative' to understand the behavior. ill modify accordingly.